php/WP_CLI/UpgraderSkin.php

```diff
@@ -41,7 +41,9 @@ function feedback( $string ) {
 
 		$string = str_replace( '&#8230;', '...', strip_tags( $string ) );
 
-		\WP_CLI::line( $string );
+		if ( empty( $this->upgrader->cli_output_format ) || ! in_array( $this->upgrader->cli_output_format, array( 'csv', 'json' ) ) ) {
+			\WP_CLI::line( $string );
```

Comment:
If we use `\WP_CLI::log()` here, we won't need to assign `$this->upgrader->cli_output_format`